geode-core/src/main/java/org/apache/geode/internal/logging/LogWriterFactory.java

```diff
@@ -87,19 +87,19 @@ public static InternalLogWriter createLogWriterLogger(final boolean isLoner,
         // LOG:CONFIG:
         logger.info(LogMarker.CONFIG, Banner.getString(null));
       }
+      System.setProperty(InternalLocator.INHIBIT_DM_BANNER, "true"); // Ensure no more banners will
```

Comment:
I think we need to find a way to do this without setting INHIBIT_DM_BANNER. If the user closes the Cache and reopens it then this system property will still be true.